tests/s3_s3express_client_test.c

```diff
@@ -256,14 +256,102 @@ static int s_s3express_client_put_test_real_server_helper(struct aws_allocator *
     return AWS_OP_SUCCESS;
 }
 
-TEST_CASE(s3express_client_put_test_small_real_server) {
+TEST_CASE(s3express_client_put_test_small) {
     (void)ctx;
-    return s_s3express_client_put_test_real_server_helper(allocator, MB_TO_BYTES(1));
+    return s_s3express_client_put_test_helper(allocator, MB_TO_BYTES(1));
 }
 
-TEST_CASE(s3express_client_put_test_large_real_server) {
+TEST_CASE(s3express_client_put_test_large) {
     (void)ctx;
-    return s_s3express_client_put_test_real_server_helper(allocator, MB_TO_BYTES(100));
+    return s_s3express_client_put_test_helper(allocator, MB_TO_BYTES(100));
+}
+
+TEST_CASE(s3express_client_put_test_multiple) {
+    (void)ctx;
+
+#define NUM_REQUESTS 100
```

Comment:
cant this just be a local enum var as a work around for c99 not allowing const size_t to be size of array?